src/beast/beast/utility/Journal.h

```diff
@@ -70,6 +78,7 @@ class Journal
         Sink () = delete;
         explicit Sink(Sink const& sink) = default;
         Sink (Severity thresh, bool console);
+        Sink& operator= (Sink& lhs) = delete;
```

Comment:
Need a `const` on the param: I.e. `Sink& operator=(Sink const&) = delete` (what's here will still give an error if someone tried assign a `const sink` to a sink, but not the one we want).